src/Helper/PriceHelper.php

```diff
@@ -49,6 +49,6 @@
      */
     public static function format($price)
     {
-        return is_numeric($price) ? number_format($price, 2, '.', '') : null;
+        return is_numeric($price) ? number_format($price, 2, '.', '') : '';
```

Comment:
This is a bit scary change. Let's fix the annotation

Reply:
Reverted and return annotation changed